misc/anonymization.md

```diff
@@ -0,0 +1,29 @@
+# Model: `chcemvediet.apps.anonymization`
+
+![](assets/anonymization.svg)
+
+## `AttachmentNormalization`
+
+Represents a single local file, which was created by normalization of Attachment file.
+
+Relations:
+* `attachment`: Attachment; May NOT be NULL.
+The Attachment from which AttachmentNormalization was created.
+
+Properties:
+* `successful`: Boolean; May NOT be NULL.
+* `file`: File; May NOT be NULL.
+Empty filename if normalization failed or normalization didn’t create any file.
```

Comment:
@viliambalaz Vlozit line-break

Reply:
:ok:

Reply:
@viliambalaz Ako sa ti tam podarilo ten linebreak vlozit? Vyrendruje to pekne zalemene:
![linebreak](https://user-images.githubusercontent.com/5547275/62835975-8ffa3d00-bc5e-11e9-9654-81d1e07e0838.png)

Ale ked riadok skusim skopirovat, tak nie je zalomeny. Co to je za magiu?

Reply:
Pouzil som 2 medzery. Akym sposobom mam teda zalomit riadok? Pomocou `<br/>`, `\`, alebo `&nbsp;`

Reply:
Ok. Som ani nevedel, ze dve medzery funguju. Ked som to editoval rucne na webe githubu, tak to nefungovalo. Sice sa mi nepaci, ze trailing medzery maju v MD vyznam, dokumentacia MD to vyslovene hovori a odporuca pouzivat namiesto `<br/>`. Tak teda pouzivajme dve trailing medzery.

Pls analogicky (dvomi traling medzerami) to zalom aj za `attachment` fieldom. A tiez v `attachments.md` ako bolo povodne zalomene v `attachments.odt`. Ked som robil `attachments.md` tak som to nezalomil, lebo som nevedel ako.

Reply:
:ok: